src/sagemaker/jumpstart/curated_hub/types.py

```diff
@@ -19,6 +19,16 @@
 
 from sagemaker.jumpstart.types import JumpStartDataHolderType, JumpStartModelSpecs
 
+class CuratedHubTagName(str, Enum):
```

Comment:
Let's come up with a better enum name than this. How about `CuratedHubUnsupportedFlag`?

Reply:
IMO this name gives flexibility to tag with other values other than `Unsupported` values. I would personally vote to keep this as is

Reply:
Hmm I still disagree. This enum should only be for `UnsupportedFlag`s since it's trying to perform a specific job. If you want to make enums for all tags for CuratedHub models, you can create a new enum, then have a aggregator type

```
class CuratedHubUnsupportedFlag(str, Enum):
  ...

class CuratedHubBlahBlah(str, Enum):
  ...

CuratedHubTag = CuratedHubUnsupportedFlag | CuratedHubBlahBlah
```

I'd prefer if we didn't overthink this to be generalizable, and cross that bridge when we get there. This is a private beta, and we need to keep the logic rigid.